nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Aj user posts

Details:
Added User Details page. 
Fixes: #24 
Testing Instructions:
Go to userprofiles page and click the users name,
this will take you to the user details page.
Users information should be displayed 
[X] New Feature
[X] Comments added to hard to understand code.
